### ISSUE TITLE
[Accessibility]: Search results: Need to tab through the whole sidebar to get to search results

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#34722]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34190

### Problem description

On the search results page, focus starts on the search box, so the user needs to tab through the whole sidebar before getting to the search results.

### Expected behavior

We probably want a visually-hidden skip-to-content button for keyboard users

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
